libbpfgo/libbpfgo.go

```diff
@@ -1,5 +1,16 @@
 package libbpfgo
 
+import "C"
```

Comment:
Any reason to move all the imports here?
Import "c" should be after the c code (and from all the cgo examples I saw, all the other imports come after it)